pendingGuardianRemover/go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/multiversx/mx-chain-tools-go/pendingGuardianRemover
+
+go 1.18
+
+require (
+	github.com/multiversx/mx-chain-core-go v1.1.34-0.20230308105332-12c26500c7f7
```

Comment:
maybe proper release? Or later?

Reply:
proper release will come once we have one for feat/freeze-account